webpack.config.js

```diff
@@ -77,6 +77,7 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env': {
         NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+        DPDASH_VERSION: JSON.stringify(process.env.npm_package_version),
```

Comment:
is this supposed to be the version in the package.json? If so can we use it directly?

Reply:
yes this is the package version from the package.json. Can we use it directly in the project..? yes, have to read it from the env variable from node and then pass that variable to the frontend

Reply:
can we read it directly from the package.json though, instead of an env variable?